tldr.py

```diff
@@ -161,8 +161,7 @@ def get_page(command, platform=None):
             if e.code != 404:
                 raise
 
-    print(command + " documentation is not available\n"
-          "Consider contributing Pull Request to https://github.com/tldr-pages/tldr")
+    return False
```

Comment:
I would suggest returning empty array here. Because `get_page_for_platform` returns the array of lines, so logically it makes sense to do something similar.

Reply:
Empty array can mean an empty page. So I think something like False or None is better.

Reply:
As I see it, either `get_page_for_platform` succeeds and returns the page, or it fails and raises an exception. So `return False` will never be executed. So probably `return None` makes more sense.

Reply:
No, it's executed when 404 error occurs.
`get_page_for_platform` raises exception, but get_page catches it (`if e.code != 404: raise`)